pkg/resources/aws.go

```diff
@@ -2047,3 +2048,46 @@ func FindELBName(tags []*elb.Tag) string {
 	}
 	return ""
 }
+
+// FilterUserProvidedItems removes user provided items such as subnets, and or the vpc.
+func FilterUserProvidedItems(resourceTrackers map[string]*ResourceTracker, clusterSpec api.ClusterSpec) {
+	for resourceTrackerKey, resourceTrackerValue := range resourceTrackers {
```

Comment:
Don't love the naming of`resourceTrackerKey,  resourceTrackerValue`. Would be great if we can name the key, value based on what they actually represent. Maybe:
```
for resourceName, resourceTracker := range resourceTrackers
```
Or other variation that represents the actual values better :)